src/single-spa-react.js

```diff
@@ -87,11 +92,7 @@ function mount(opts, props) {
 
     const rootComponentElement = opts.React.createElement(opts.rootComponent, props)
     const elementToRender = SingleSpaContext ? opts.React.createElement(SingleSpaContext.Provider, {value: props}, rootComponentElement) : rootComponentElement
-
     const renderedComponent = opts.ReactDOM.render(elementToRender, getRootDomEl(domElementGetter), whenFinished)
-    if (!renderedComponent.componentDidCatch && !opts.suppressComponentDidCatchWarning && atLeastReact16(opts.React)) {
```

Comment:
I'm assuming something changed but renderedComponent wasn't providing componentDidCatch anymore. Instead I'm inspecting the root component's prototype